extension/styles.css

```diff
@@ -0,0 +1,12 @@
+body {
```

Comment:
css classnames cost almost nothing. The cost of the classnames is not the issue. The cost of a classname is an extremely minute optimization. What is more important here is code modularity and flexibility. You only get one body but you get as many classnames as you want. Get our of your own way by naming your styles instead of locking yourself into one body style.